src/services/HyperClient.ts

```diff
@@ -85,6 +85,27 @@ export default class HyperClient {
     return ip
   }
 
+  async getPodList (): Promise<Array<string>> {
+    if (this.config.noop) return ['']
+    const res = await axios.request({
+      socketPath: this.config.hyperSock,
+      method: 'get',
+      url: '/list?item=pod',
+      responseType: 'json'
+    })
+
+    const pods = res.data.podData.reduce((acc: Array<string>, curr: string) => {
```

Comment:
This could be a filter instead of a reduce

Reply:
Added comment and changed the name of curr to be more descriptive. We have to take what we get back from hyper list and pull just the hash and its status to make an array of just running pod manifest hashes to compare against our leveldb.